web-client/src/pages/Home/ClaimControls.vue

```diff
@@ -146,7 +160,7 @@ watch(client.isConnected, async (isConnected) => {
                     >
                         <option :value="null" />
                         <option
-                            v-for="token in availableTokens"
+                            v-for="token in availableTokens.map( available => available.tokenId )"
```

Comment:
I think this should be equivalent
 
```suggestion
                            v-for="{ tokenId, minted } in availableTokens"
```

We could then also use the `minted` parameter to indicate whether the token is minted directly in the select option, ie `Already minted - 123`

Reply:
Totally agree, this helps in showing the minted status without selecting it 👍🏻👍🏻